### PR TITLE
chore: Keep Rajat as codeowner for only cypress tests of widgets

### PR DESCRIPTION
**Pull request by rajatagrawal**
Keep @rajatagrawal as code owner for only cypress tests of widgets.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -23,7 +23,6 @@ app/client/tsconfig.json @KelvinOm
 app/client/tsconfig.path.json @KelvinOm
 
 # App viewers pod
-app/client/src/widgets/* @appsmithorg/app-viewers @rajatagrawal
```
Comment (somangshu) [inline CODEOWNERS:26]: why should we  not tag app-viewers is something in the widgets folder changes? Am I getting this incorrectly?

Comment (rajatagrawal) [reply CODEOWNERS:26]: Done

**Review by somangshu — APPROVED** [review]
LGTM

**Comment by somangshu** [comment]
LMK Is we can merge this off

**Comment by rajatagrawal** [comment]
@somangshu Yes, we can merge it. Thank you!

**Pull request merged**